src/gridtools/clang/benchmark_writer.hpp

```diff
@@ -0,0 +1,146 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                         _       _
+//                        | |     | |
+//                    __ _| |_ ___| | __ _ _ __   __ _
+//                   / _` | __/ __| |/ _` | '_ \ / _` |
+//                  | (_| | || (__| | (_| | | | | (_| |
+//                   \__, |\__\___|_|\__,_|_| |_|\__, | - GridTools Clang DSL
+//                    __/ |                       __/ |
+//                   |___/                       |___/
+//
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+
+#ifndef GRIDTOOLS_CLANG_BENCHMARK_WRITER_HPP
+#define GRIDTOOLS_CLANG_BENCHMARK_WRITER_HPP
+
+#include "gridtools/clang/Support/External/json/json.hpp"
+#include "gridtools/clang_dsl.hpp"
+#include <chrono>
+#include <ctime>
+#include <fstream>
+#include <vector>
+
+namespace gridtools {
+
+namespace clang {
+
+class benchmarker {
```

Comment:
we are not using this, are we?

Reply:
we should make the change in the dycore once we have it here